docs/sources/alerting/configure-notifications/template-notifications/images-in-notifications.md

```diff
@@ -1,4 +1,6 @@
 ---
+aliases:
+  - ../manage-notifications/images-in-notifications/ # /docs/grafana/latest/alerting/manage-notifications/images-in-notifications/
```

Comment:
This URL is already redirected to `/docs/grafana/latest/alerting/configure-notifications/template-notifications/` in https://github.com/grafana/grafana/pull/83372/files#diff-7b84f73c1142fd039b1d6f3f6f4611f464df76936e8fb4cc77f163a6248faffcR3.

Given the title of this page, I think that other redirect should be removed.

Reply:
@jdbaldry not sure what you mean by this one? leave just this one redirect?

Reply:
yeah I think this one should stay but I wanted to point out where the other alias redirecting that page lived so you could decide which is better.